docs/docs/configuration/enum.mdx

```diff
@@ -21,14 +21,24 @@ Available strategies:
 
 The `IgnoreCase` property allows to opt in for case insensitive mappings (defaults to `false`).
 
+Enum from/to strings mappings can be customized by setting the enum naming strategy to be used.
+You can specify the naming strategy using `NamingStrategy` in `MapEnumAttribute` or `MapperAttribute`.
+Available naming strategies:
+
+| Name       | Description                                     |
+| -----------| ------------------------------------------------|
+| PascalCase | Matches enum values using PascalCase. (default) |
```

Comment:
Wouldn't another default `MemberName` or `Default` (or similar) be needed? The default just uses the member name of the c# enum member, this might be PascalCase but doesn't have to. We should probably just rename the default.

Reply:
Yes, agreed. I have added MemberName.